src/test/java/org/beanpod/switchboard/controller/DeviceControllerTest.java

```diff
@@ -52,6 +53,19 @@ void setup() {
     MockitoAnnotations.initMocks(this);
   }
 
+  @Test
+  final void testUploadJson() {
```

Comment:
Rename, since we aren't doing exclusively json anymore

Reply:
fixed